src/features/device-logs/lib/backends/loki.ts

```diff
@@ -400,12 +400,14 @@ export class LokiBackend implements DeviceLogsBackend {
 	) {
 		return logs.map((log) => {
 			this.validateLog(log);
-			log.version = VERSION;
 			const timestamp = new loki.Timestamp();
 			timestamp.setSeconds(Math.floor(Number(log.nanoTimestamp / 1000000000n)));
 			timestamp.setNanos(Number(log.nanoTimestamp % 1000000000n));
 			// store log line as JSON
-			const logJson = JSON.stringify(log, omitNanoTimestamp);
+			const logJson = JSON.stringify(
+				{ ...log, version: VERSION },
```

Comment:
This change is to not mutate the log, it isn't a problem when loki is the secondary but when it's the primary it can cause problems for the secondary as they're getting a loki versioned log which they don't expect